SR2022/2021-11-10-kickstart-details.md

```diff
@@ -0,0 +1,29 @@
+---
+to: Student Robotics 2022 registered teams
+subject: Kickstart Details
+---
+
+Hi teams
+
+Only a couple of days left until Saturday’s Kickstart event, so here are some more details on what to expect and what we need from you.
+
+## Kits
+
+If you don’t already have a kit, we will have contacted you regarding the delivery arrangements. We need you to sign a kit disclaimer form asap to ensure that the kit can arrive shortly after Kickstart.
+
+If you have a kit from a previous year, we will soon be sending you instructions on how to update the software and check that everything is present and correct.
+
+## Kickstart
+
+As you already know, [Kickstart](https://studentrobotics.org/events/sr2022/kickstart/) is at 1pm uk time this Saturday 13th November. The livestream will last for approximately an hour and we will explain this year’s game, the competition structure, and give an overview of the kit.
+
+As we are not running a Kickstart in a centralised location, we have prepared an activity as a kick-off session for your team to learn their robotics kit. We'll be sending it out to you immediately after Kickstart, but we can also optionally provide virtual/physical support. If you’re interested, please [sign up here](https://forms.gle/ojysPbkfcpKCtW8z6).
```

Comment:
Suggestion: Assuming it's the case, we should mention here that the kit isn't required for these "kick-off activities" (microgames).

Reply:
The kit is required, step 1 is upgrade the kit, step 2 is plug it together